### PR TITLE
Add content types for .bval and .bvec

### PR DESCRIPTION
**Pull request by eapapp**

### CODE REVIEW

**Comment by UlrikeS91** [comment]
Could somebody with more knowledge about both content types and neuroimaging review this PR? 

@lzehl, maybe @apdavison?

**Comment by apdavison** [comment]
@Peyman-N can you review this, since it seems to be for MRI data?

**Review by lzehl — CHANGES_REQUESTED** [review]
I think both content types are defined by the FSL library (https://fsl.fmrib.ox.ac.uk/fsl/fslwiki/FDT/UserGuide). Therefore I would suggest the following changes:

"name": "application/vnd.fsl.bval.text"

"name": "application/vnd.fsl.bvec.text"

Maybe add short description for both? Example adapted from the FSL user guide (@Peyman-N and @eapapp feel free to further optimize):

The FSL bval format is used for ASCII text file that contains a list of b-values applied during each volume acquisition. The b-values are assumed to be in s/mm^2 units. The order of entries in this file must match the order of volumes in the diffusion weighted input  data (4D NIFTI file) and entries in the gradient directions text file (FSL bvec format). 

The FSL bvec format is used for ASCII text file that contains a list of gradient directions applied during diffusion weighted volumes. The order of entries in this file must match the order of volumes in the input data (4D NIFTI file).

**Comment by eapapp** [comment]
Hi Lyuba,

Do we need the ".text" part at the end of the name? Looking at other similar schema names, I was wondering if application/vnd.fsl.bval and application/vnd.fsl.bvec would be enough, since I don't see the ending .text in other names. I'm asking because we need to rename the files as well, I suppose (text_x-bval.jsonld -> application_vnd.fsl.bval.jsonld if I omit the .text part).

What do you think?

All the best,
Eszter

**Comment by lzehl** [comment]
@eapapp I took this part from IANA as example where often the base format (in this case text) is indicated in the name. @apdavison what do you think? I find these indications useful but I'm not sure I fully understand the IANA naming convention there.

**Comment by lzehl** [comment]
@apdavison thank you. @eapapp then let's update to "text/vnd.fsl.bval" / "text/vnd.fsl.bvec" (including updating the filenames to "text_vnd.fsl.bval.jsonld" / "text_vnd.fsl.bvec.jsonld")

**Comment by lzehl** [comment]
@Peyman-N would you please review the changes I made to support the merge of @eapapp PR? If you think this is good please approve and merge

**Review by lzehl — APPROVED** [review]

**Comment by eapapp** [comment]
Hi @Peyman-N, could you please review and merge this PR if it looks okay? It would be great to have these two content types available in the KG. Thanks a lot! 🙏🏻

File: instances/latest/contentTypes/text_vnd.fsl.bvec.jsonld [hunk 1]
```diff
@@ -0,0 +1,20 @@
+{
+  "@context": {
+    "@vocab": "https://openminds.ebrains.eu/vocab/"
+  },
+  "@id": "https://openminds.ebrains.eu/instances/contentTypes/text_vnd.fsl.bvec",
+  "@type": "https://openminds.ebrains.eu/core/ContentType",
+  "dataType": null,
+  "description": "The FSL bvec format is used for ASCII text file that contains a list of gradient directions applied during diffusion weighted volumes. The order of entries in this file must match the order of volumes in the input data (4D NIFTI file).",
```
Comment (Peyman-N) [inline instances/latest/contentTypes/text_vnd.fsl.bvec.jsonld:8]: It is essential for this type to specify that the file contains three rows: the first row for the x-axis, the second row for the y-axis, and the third row for the z-axis.

**Comment by Peyman-N** [comment]
Hi @eapapp Sorry I don't know why I didn't get any notification for this. 
Just a small modification and I think it is good to go.

**Comment by eapapp** [comment]
Hi @Peyman-N , I have updated the description. Please let me know if there is anything more I can do (you can also edit the files directly if you like).

**Review by Peyman-N — APPROVED** [review]

**Pull request merged**